woo-tracking-jsdoc/publish.js

```diff
@@ -0,0 +1,77 @@
+const fs = require( 'jsdoc/fs' );
+const env = require( 'jsdoc/env' );
+const path = require( 'jsdoc/path' );
+
+// RegExp used to match the replacement. The groups are respectively: start marker, replacable content, end marker.
+const defaultReplacementRegex = /(<woo-tracking-jsdoc.*>)([\s\S]*)(<\/woo-tracking-jsdoc.*>)/;
+
+const disclaimer = `<!---
+Everything inside \`<woo-tracking-jsdoc>\` will be automatically generated by \`woo-tracking-jsdoc\`.
+Do not edit its content manually!
+-->`;
+
+/** @module publish */
+
+/**
+ * Generate documentation output.
+ *
+ * @param {Object} data - A TaffyDB collection representing
+ *                       all the symbols documented in your code.
+ */
+exports.publish = function ( data ) {
+	const {
+		conf: {
+			templates: {
+				'woo-tracking-jsdoc': {
+					path: readmePath = 'TRACKING.md',
+					replacement,
+				} = {},
+			} = {},
+		} = {},
+		pwd,
+	} = env;
+
+	const replacementRegex = replacement
+		? new RegExp( replacement )
+		: defaultReplacementRegex;
+
+	let mdResult = '\n';
+
+	data( { kind: 'event' } ).each( ( symbol ) => {
```

Comment:
💅 
Maybe add a new line to separate each tracking event block?

![image](https://user-images.githubusercontent.com/17420811/140302954-7e1c02b3-63c8-4e9e-ba38-9eef12f58af7.png)

Reply:
Addressed in https://github.com/woocommerce/pinterest-for-woocommerce/pull/238/commits/59194d8d1cd173e288bffbdd5f5015bddf6d4ac6